### ISSUE TITLE
How to update different part of model using different loss?

### ISSUE THREAD

**Issue by tankche1** [open-mmlab/mmdetection#2158]
Let's consider a simple case.
```
model = {
      model.fc1 = nn.Linear(10,1)
      model.fc2 = nn.Linear(10,1)
}

optimizer1 = SGD(model.fc1)
optimizer2 = Adam(model.fc2)

optimizer1.zero_grad()
loss1 = Loss_1(model.fc1(x))
loss1.backward()
optimizer1.step()

optimizer2.zero_grad()
loss2 = Loss_2(model.fc2(x))
loss2.backward()
optimizer2.step()
```
Any way to do this? The code is based on mmcv.runner. Not sure how to do that.

**Comment by ZwwWayne** [comment]
For now, it is difficult to achieve your goal; one way could do this is to implement a new optimizer that contains many optimizers.
This feature is in our plan, and we will enable that in the following several months.
